tests/test_traptor_offline.py

```diff
@@ -323,4 +324,17 @@ def test_ensure_traptor_only_enriches_tweets(self, traptor, non_tweet_stream_mes
         for message in non_tweet_stream_messages:
             print(message)
             enriched_data = traptor._enrich_tweet(message)
-            assert enriched_data == message
\ No newline at end of file
+            assert enriched_data == message
+
+    def test_ensure_traptor_stays_alive_until_rules_are_found(self, traptor):
```

Comment:
Can you explain how this tests the method?  I'm still trying to wrap my head around MagicMock, Pytest, etc.